### DISCUSSION TITLE
Allow global css imports in well-behaved node_modules

### DISCUSSION THREAD

**Discussion by redallen** [vercel/next.js#25094] [Ideas]
### What version of Next.js are you using?

10.2.0

### What version of Node.js are you using?

14.16.0

### What browser are you using?

Chrome

### What operating system are you using?

Linux

### How are you deploying your application?

next build

### Describe the Bug

Importing global css in `node_modules` should not break for well-behaved libraries like `patternfly` which scope their css to `pf-*`.

### Expected Behavior

`next build` to not throw this error:
```sh
./node_modules/@patternfly/react-styles/css/components/AboutModalBox/about-modal-box.css
Global CSS cannot be imported from within node_modules.
Read more: https://nextjs.org/docs/messages/css-npm
Location: node_modules/@patternfly/react-styles/css/components/AboutModalBox/about-modal-box.js
```

### To Reproduce

`npx create-next-app`

Add `import { Button } from '@patternfly/react-core'` and a `<Button />` to `next-app/pages.index.js`

Related: #4035

**Comment by timneutkens** [comment]
Opened a RFC here: https://github.com/vercel/next.js/discussions/27953
